File: 240119/폭탄 해제/bomb-removal.py
class Heje:
    def __init__(self, code=0, color=0, sec=0):
        self.code = code
        self.color = color
        self.sec = sec


code, color, sec = input().split()
heje1 = Heje(code, color, int(sec))
print("code :", heje1.code)
print("color :", heje1.color)
print("second :", heje1.sec)

File: 240424/폭탄 해제/bomb-removal.cpp
#include <iostream>
#include <string>
using namespace std;

class Bomb {
    public:
        string code;
        char color;
        int second;

        Bomb(string code="", char color='\0', int second=0){
            this->code = code;
            this->color = color;
            this->second = second;
        }
};

int main() {
    Bomb b=Bomb();
    cin >> b.code >> b.color >> b.second;
    cout << "code : " << b.code << endl
         << "color : " << b.color << endl
         << "second : " << b.second;
    return 0;
}